src/controllers/livetv/livetvsuggested.js

```diff
@@ -1,393 +1,402 @@
-            return new Date().getTime() - lastFullRender > 3e5;
-        }
+    function onTabChange(evt) {
+        const previousTabController = tabControllers[parseInt(evt.detail.previousIndex)];
 
-        function onBeforeTabChange(evt) {
-            preLoadTab(view, parseInt(evt.detail.selectedTabIndex));
+        if (previousTabController && previousTabController.onHide) {
+            previousTabController.onHide();
         }
 
-        function onTabChange(evt) {
-            var previousTabController = tabControllers[parseInt(evt.detail.previousIndex)];
+        loadTab(view, parseInt(evt.detail.selectedTabIndex));
+    }
 
-            if (previousTabController && previousTabController.onHide) {
-                previousTabController.onHide();
-            }
+    function getTabContainers() {
+        return view.querySelectorAll('.pageTabContent');
+    }
 
-            loadTab(view, parseInt(evt.detail.selectedTabIndex));
-        }
+    function initTabs() {
+        mainTabsManager.setTabs(view, currentTabIndex, getTabs, getTabContainers, onBeforeTabChange, onTabChange);
+    }
 
-        function getTabContainers() {
-            return view.querySelectorAll('.pageTabContent');
-        }
+    function getTabController(page, index, callback) {
+        const depends = [];
 
-        function initTabs() {
-            mainTabsManager.setTabs(view, currentTabIndex, getTabs, getTabContainers, onBeforeTabChange, onTabChange);
-        }
+        // TODO int is a little hard to read
+        switch (index) {
+            case 0:
+                break;
 
-        function getTabController(page, index, callback) {
-            var depends = [];
+            case 1:
+                depends.push('controllers/livetv/livetvguide');
+                break;
 
-            // TODO int is a little hard to read
-            switch (index) {
-                case 0:
-                    break;
+            case 2:
+                depends.push('controllers/livetv/livetvchannels');
+                break;
 
-                case 1:
-                    depends.push('controllers/livetv/livetvguide');
-                    break;
+            case 3:
+                depends.push('controllers/livetv/livetvrecordings');
+                break;
 
-                case 2:
-                    depends.push('controllers/livetv/livetvchannels');
-                    break;
+            case 4:
+                depends.push('controllers/livetv/livetvschedule');
+                break;
 
-                case 3:
-                    depends.push('controllers/livetv/livetvrecordings');
-                    break;
+            case 5:
+                depends.push('controllers/livetv/livetvseriestimers');
+                break;
 
-                case 4:
-                    depends.push('controllers/livetv/livetvschedule');
-                    break;
+            case 6:
+                depends.push('scripts/searchtab');
+        }
```

Comment:
This switch statement needs to be converted from
`depends.push` -> `depends = 'path'`
since the import does not import as an array but as a string
see [here](https://github.com/jellyfin/jellyfin-web/blob/master/src/controllers/music/musicrecommended.js#L273-L302)

Reply:
Yeah makes sense